crates/connectors/ndc-postgres/src/query.rs

```diff
@@ -93,10 +93,6 @@ async fn execute_query(
                 tracing::error!("{}", err);
                 // log error metric
                 match &err {
-                    query_engine_execution::query::QueryError::ReservedVariableName(_) => {
```

Comment:
There are no longer any occasions that can result in a variable name being reserved.